votingpool/db.go

```diff
@@ -70,7 +139,7 @@ func putPool(tx walletdb.Tx, votingPoolID []byte) error {
 // loadAllSeries returns a map of all the series stored inside a voting pool
 // bucket, keyed by id.
 func loadAllSeries(tx walletdb.Tx, votingPoolID []byte) (map[uint32]*dbSeriesRow, error) {
```

Comment:
Actually here it shouldn't be a problem because the plan is to have only one series per week. Having said that, I think we should get rid of loadAllSeries completely because we have nothing that operates on the full set of series. Right now loadAllSeries() populates .seriesLookup which is then used by Pool.GetSeries() but we could just make it load the desired series directly from the DB, and get rid of both loadAllSeries() and seriesLookup.